src/sestest/CommandLineTokenPropertyProvider.cs

```diff
@@ -22,7 +22,7 @@ public class CommandLineEntitlementPropertyProvider : IEntitlementPropertyProvid
         // A steady reference for "now"
         private readonly DateTimeOffset _now = DateTimeOffset.Now;
 
-        private readonly string _entitlementId = $"entitlement-{Guid.NewGuid():D}";
+        private readonly string _tokenId = $"entitlement-{Guid.NewGuid():D}";
```

Comment:
Since this is the `id` for the token, not the entitlement, perhaps a different prefix:

``` csharp
private readonly string _tokenId = $"token-{Guid.NewGuid():D}";
```